### ISSUE TITLE
The issue of development

### ISSUE THREAD

**Issue by SPAWNRYS-ban** [bundabrg/GeyserReversion#65]
Hi. I would like to ask if the project will be further developed? I need something like this as many players on my server play from different versions.

And will it be on Velocity (a better analogue of Bungeecord)?

**Comment by maxfire2008** [comment]
See #64 #59 for more info.

As for Velocity I guess one will be able to run it separately like so: Player on EE --> Reversion --> Geyser --> Velocity
But don't quote me on that

**Comment by SPAWNRYS-ban** [comment]
Yeah... So I'll have to mess around with floodgate and LimboAUTH to make it work properly.

**Comment by CamperSamu** [comment]
Glad you're planning on suppporting LimboAuth!
